cache/FIFO_FH.h

```diff
@@ -368,6 +379,9 @@ bool FIFO_FHCache<TKey, TValue, THash>::construct_ratio(double FC_ratio) {
     
     // Node exists
     m_fasthash->insert(temp_node->m_key, temp_hashAccessor->second.m_value);
+    m_mapMutex.lock();
+    chunkMapper[temp_node->m_key] = chunkCounter;
+    m_mapMutex.unlock();
```

Comment:
Wait if we do end up needing to lock during hash map insertion, then would it be better to use CLHT (or maybe TBB)?